### PR TITLE
Fix: Opening Accordion Block in Theme Customizer

### LINKED ISSUE #2956: Accordion block not opening on Theme Customizer > Footer & Header / Widgets
**Describe the bug**
- This issue applies on themes with customizer such as Blocksy, 2021, Astra, etc.

**To Reproduce**
1. Activate 2021 theme
2. Go to Appearance > Customize
3. Go to Widgets
4. Add an Accordion block
5. Try expanding the accordion
6. See that it doesn't expand

**Expected behavior**
Should be able to expand Accordion block on Theme Customizer > Widgets

**Screenshots**

https://github.com/gambitph/Stackable/assets/103395655/88e1d4c1-7daf-4e3d-9e5b-86beb1de04fd

### PR DESCRIPTION
**Pull request by kaeizen**
fixes #2956

### CODE REVIEW

File: src/block/accordion/edit.js [hunk 1]
```diff
@@ -73,12 +73,17 @@ const Edit = props => {
 
 	const blockAlignmentClass = getAlignmentClasses( props.attributes )
 
+	// getEditorDom() always returns null in theme customizer, so we need to manually get its editor
```
Comment (bfintal) [inline src/block/accordion/edit.js]:
```suggestion
	// Hacky fix: getEditorDom() always returns null in theme customizer, so we need to manually get its editor
```

File: src/block/accordion/edit.js [hunk 2]
```diff
@@ -73,12 +73,17 @@ const Edit = props => {
 
 	const blockAlignmentClass = getAlignmentClasses( props.attributes )
 
+	// Hacky fix: getEditorDom() always returns null in theme customizer, so we need to manually get its editor
+	const isCustomizer = window.location.href && window.location.href.indexOf( 'customize.php' ) !== -1
```
Comment (bfintal) [inline src/block/accordion/edit.js]:
```suggestion
	const isCustomizer = window?.location?.href && window.location.href.indexOf( 'customize.php' ) !== -1
```

File: src/block/accordion/edit.js [hunk 3]
```diff
@@ -101,7 +106,7 @@ const Edit = props => {
 
 	// When first adding an accordion, the inner blocks may not be rendered yet, wait for it.
 	if ( ! hasInitClickHandler ) {
-		const headerEl = getEditorDom()?.querySelector( `[data-block="${ clientId }"] [data-type="stackable/column"]` )
+		const headerEl = getDom?.querySelector( `[data-block="${ clientId }"] [data-type="stackable/column"]` )
```
Comment (bfintal) [inline src/block/accordion/edit.js]:
```suggestion
		const headerEl = getDom()?.querySelector( `[data-block="${ clientId }"] [data-type="stackable/column"]` )
```

**Comment by andeng1106** [comment]
@mxkae Remaining issues:

- Issue is now fixed for 2021 theme but still happens on other themes. Here are some themes tested (make sure to include fix for other themes with customizer that are not mentioned)).
- Accordion doesn't open in:
    - Blocksy
         - Theme Customizer > Footer 
         - Widgets > Footer Widget Area (any)
    - Astra
         - Widgets > Header & Footer Bar Section (any)
    - OceanWP
         - Theme Customizer > Widgets > Footer 
         - Widgets > Footer (any)
    - GeneratePress
         - Theme Customizer > Widgets
         - Widgets

Here is a sample using Blocksy theme (customizer & widgets):

https://github.com/gambitph/Stackable/assets/103395655/ff6e55ae-6eef-45b6-ac88-27e3ad93ab67

**Pull request merged**